skt/executable.py

```diff
@@ -36,6 +36,7 @@
 import skt.runner
 from skt.kernelbuilder import KernelBuilder, CommandTimeoutError, ParsingError
 from skt.kerneltree import KernelTree, PatchApplicationError
+from skt.misc import join_with_slash, get_patch_name, get_patch_mbox
```

Comment:
Can you use a single import statement with parenthesis, as PEP328 suggests, instead of using two skt.misc import lines?